source/stylesheets/_circle.scss

```diff
@@ -62,11 +62,51 @@
   }
 }
 
+.circle-members-header {
+  text-align: center;
+  .circle-logo {
+    max-width: 60%;
+  }
+}
+
+.circle-members {
+  padding: 1em 0;
+  .circle-list-wrapper {
+    margin-bottom: 1em;
+    text-align: center;
+  }
+
+  .circle-title {
+    text-align: center;
+    text-transform: uppercase;
+    letter-spacing: 3px;
+    padding: .5em;
+    font-size: 1.5em;
+  }
+
+  .circle-list {
+    font-size: 1em;
+    line-height: 1.5em;
+  }
+
+  .yellow-star {
+    color: $yellow-tribune;
+    font-size: .8em;
+  }
+}
+
 
 .benefits-header {
   padding: 1em 0 0.5em 0;
 }
 
+.benefits-list {
```

Comment:
Feels like the spacing is off on this. A margin: 0 on unordered lists might be a good thing to add, and I think that would address the spacing. However, my other concern is that making this a ul here makes this inconsistent with the way benefits are listed in the right column on the homepage of the app. It should probably be either all paragraphs, or the benefits on the homepage should be made into unordered lists. I'm less concerned about which it is than that it's consistent across pages.